src/state/user/reducer.ts

```diff
@@ -235,23 +227,16 @@ export default createReducer(initialState, builder =>
       state.popup = { success: false, message: action.error.message || 'Error occured while saving NFT to drafts' }
     })
     .addCase(getUserNFTsAction.pending, (state, action) => {
-      // console.log('pending')
+      console.log('pending')
     })
     .addCase(getUserNFTsAction.fulfilled, (state, action) => {
+      console.log('reducer',action.payload)
       state.onMarket = action.payload.onMarket
       state.offMarket = action.payload.offMarket
       state.userAuctions = action.payload.userAuctions
     })
     .addCase(getUserNFTsAction.rejected, (state, action) => {
+      console.log('error',action)
       state.popup = { success: false, message: action.error.message || 'Error occured while saving NFT to drafts' }
```

Comment:
should load from localization

Reply:
this is from the old inventory I comment all out